include/swift/AST/SearchPathOptions.h

```diff
@@ -355,27 +367,6 @@ class SearchPathOptions {
     Lookup.searchPathsDidChange();
   }
 
-  void setCompilerPluginLibraryPaths(
-      std::vector<std::string> NewCompilerPluginLibraryPaths) {
-    CompilerPluginLibraryPaths = NewCompilerPluginLibraryPaths;
-    Lookup.searchPathsDidChange();
```

Comment:
Plugin search doesn't use `Lookup`, and adding plugin search path won't affect `import` module search path. So we don't need to call `Lookup.searchPathsDidChange()` when updating `PluginSearchOpts`.